Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml

```diff
@@ -172,6 +172,22 @@
 		</Target>
 	</ECRelationshipClass>
 
+	<ECEntityClass typeName="Catchment" modifier="Sealed" displayLabel="Catchment">
+		<BaseClass>anlyt:AnalyticalElement</BaseClass>
+		<ECProperty propertyName="Area" typeName="double" displayLabel="Area" category="HydraulicData" kindOfQuantity="rru:AREA"/>
+		<ECProperty propertyName="Tc" typeName="double" displayLabel="Time of Concentration" category="HydraulicData" kindOfQuantity="AECU:TIME"/>
+		<ECProperty propertyName="RationalC" typeName="double" displayLabel="Rational C" category="HydraulicData"/>
+    <ECNavigationProperty propertyName="OutflowNode" relationshipName="CatchmentDischargesToOutflowNode" direction="Forward" />
+	</ECEntityClass>
+
+	<ECRelationshipClass typeName="CatchmentDischargesToOutflowNode" strength="referencing">
+		<Source multiplicity="(0..*)" roleLabel="discharges from" polymorphic="false">
+			<Class class="Catchment"/>
+		</Source>
+		<Target multiplicity="(0..1)" roleLabel="discharges to" polymorphic="true">
+			<Class class="INode"/>
```

Comment:
You need to specify the schema alias since INode is not a concept defined in this schema. e.g. net:INode.

Reply:
ahhhh -  derp thanks